backend/controller/authController.js

```diff
@@ -52,12 +57,35 @@ const userExist = asyncHandler(async (req, res, next) => {
 const signUp = asyncHandler(async (req, res, next) => {
   const { email, password } = req.body;
 
+  if (!email || !password) {
+    return next(
+      new customError("All fields are required. email , password", 400)
```

Comment:
Instead of this message write something like ""Email and Password are required."

Reply:
done